tools/testbench/topology.c

```diff
@@ -689,16 +689,22 @@ int parse_topology(struct sof *sof, struct shared_lib_table *library_table,
 	lib_table = library_table;
 
 	/* file size */
-	fseek(file, 0, SEEK_END);
+	if (fseek(file, 0, SEEK_END)) {
+		fprintf(stderr, "error: seek to end of topology\n");
+		return -errno;
```

Comment:
do you need to include errno.h?

Reply:
The code built OK without. I assume the headers those provided fseek and malloc included it. What's our policy with things like this?

Reply:
In general we should include headers directly if we use a functionality, unless it's _just_ an implementation detail header. As in, you wouldn't include <platform/lib/dma.h> even if something is implemented there, but you would include <sof/lib/dma.h> even if it's already included by some other header you already have, if you want to use DMA related stuff.

So you should include <errno.h> if you directly access the `errno` variable.

Reply:
OK, I'll add. It's still missing from just pushed version.

Reply:
@paulstellian It's fixed now. The other modified already had the include for errno.h.